src/Nethermind/Nethermind.JsonRpc/Data/TransactionForRpc.cs

```diff
@@ -91,7 +91,12 @@ public TransactionForRpc()
 
     public UInt256? MaxFeePerGas { get; set; }
     public long? Gas { get; set; }
-    public byte[]? Data { get; set; }
+
+    // Required for compatibility with some CLs like Prysm
+    // Accept during deserialization, ignore during serialization
+    // See: https://github.com/NethermindEth/nethermind/pull/6067
+    [JsonProperty(nameof(Data))]
+    private byte[]? Data { set { Input = value; } }
```

Comment:
The "trick" used to support deserialization but ignore serialization is described here: https://stackoverflow.com/questions/43714050/multiple-jsonproperty-name-assigned-to-single-property/43715009#43715009